packages/instantsearch.js/src/widgets/hits/__tests__/hits-integration-test.ts

```diff
@@ -122,7 +122,101 @@ describe('hits', () => {
       );
     });
 
-    it('sends click event', async () => {
+    it('sendEvent sends click event', async () => {
+      const { search } = createInstantSearch();
+      const { insights, onEvent } = createInsightsMiddlewareWithOnEvent();
+      search.use(insights);
+
+      search.addWidgets([
+        hits({
+          container,
+          templates: {
+            item: (item, { html, sendEvent }) => html`
+              <button
+                type="button"
+                onClick=${() => sendEvent('click', item, 'Item Clicked')}
+              >
+                ${item.title}
+              </button>
+            `,
+          },
+        }),
+      ]);
+      search.start();
+      await wait(0);
+
+      expect(onEvent).toHaveBeenCalledTimes(1); // view event by render
```

Comment:
Could we also test `toHaveBeenCalledWith` to test that this is a `view` event? This way we can remove the comment.

Reply:
That's already tested in the first test in this describe. It would be a little noisy IMO